src/site/content/en/metrics/inp/index.md

```diff
@@ -243,51 +186,17 @@ onINP(({value}) => {
 });
 ```
 
-Like other methods exported by `web-vitals`, `onINP` accepts a function as an argument, and will pass metric data to the function you give it. From there, you can send that data to an endpoint for collection and further analysis.
+As with other methods exported by `web-vitals`, `onINP` accepts a function as an argument, and will pass metric data to the function you give it. From there, you can send that data to an endpoint for collection and analysis.
 
 See the [`onINP()`](https://github.com/GoogleChrome/web-vitals/tree/next#oninp) reference documentation for additional usage instructions.
 
 {% Aside 'warning' %}
 Gathering INP metrics in the field will only work on browsers that [fully support the Event Timing API](https://caniuse.com/mdn-api_performanceeventtiming), including its `interactionId` property.
 {% endAside %}
 
-Measuring INP on your own in JavaScript is tricky. To get the best results, it's strongly recommended that you use the `web-vitals` JavaScript library. However, if you want to get a sense of what the longest interactions may look like on your page, you can copy and paste the JavaScript snippet below in the console of your browser's developer tools and tweak as needed.
```

Comment:
I think my slight preference would be to leave this code example in, just to give people a sense of how the Event Timing API works. But if you think it's better without it, SGTM.